docs/knative-eventing-mesh/03-01-event-processing.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Event processing and delivery
+type: Details
+---
+The event processing and delivery flow in Knative Eventing Mesh uses the Broker and Trigger concept to forward events and deliver them to the subscribers.
+This diagram explains the event flow in Kyma, from the moment the Application sends an event, to the point when the event triggers the function.
+
+![Eventing flow](./assets/eventing-mesh-flow.svg)
+
```

Comment:
could you write on the diagram Function CR (Lambda)?

Reply:
now that I look at it, you have all other CRs without "CR" on the diagram so sorry for the confusion :(